feature-libs/cart/base/components/add-to-cart/add-to-cart.component.html

```diff
@@ -18,6 +18,9 @@
       >
     </div>
   </div>
+  <ng-container *cxFeatureLevel="'5.1'">
+    <ng-template [cxOutlet]="CartOutlets.ADD_TO_CART_CONTAINER"></ng-template>
```

Comment:
`ADD_TO_CART_CONTAINER` is confusing. maybe call it `ADD_TO_CART_OPTIONS`?

Reply:
If `ADD_TO_CART_CONTAINER` is preferred, then maybe make it wrap the counter and stock info:
```
<ng-template [cxOutlet]="CartOutlets.ADD_TO_CART_CONTAINER">
    <div class="quantity" *ngIf="showQuantity">
      <label>{{ 'addToCart.quantity' | cxTranslate }}</label>
      <div class="cx-counter-stock">
        <cx-item-counter
          *ngIf="hasStock"
          [max]="maxQuantity"
          [control]="addToCartForm.get('quantity')"
        ></cx-item-counter>

        <span class="info">
          <span *ngIf="showInventory$ | async">{{ getInventory() }}</span>
          {{
            hasStock
              ? ('addToCart.inStock' | cxTranslate)
              : ('addToCart.outOfStock' | cxTranslate)
          }}</span
        >
      </div>
    </div>
  </ng-template>
```
So, in `PdpPickupOptionsContainerModule`, the `position` can be `after`. Using this way, customer can config the pickup option displayed before or after the counter.